<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatus</class>
 <widget class="QDialog" name="DialogStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>128</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>184</width>
    <height>128</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>184</width>
    <height>128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="statusOnline">
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="statusNotInPlace">
     <property name="text">
      <string>Not in place</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="statusNotDisturb">
     <property name="text">
      <string>Not Disturb</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
